contrib/test/crio-integration-results.yml

```diff
@@ -0,0 +1,66 @@
+---
+# vim-syntax: ansible
+
+- hosts: '{{ hosts | default("all") }}'
+  vars:
+    crio_integration_filepath: "/root/src/github.com/kubernetes-incubator/cri-o/testout.txt"
+    crio_node_e2e_filepath: "/root/src/k8s.io/kubernetes/node-e2e.log"
+    # Treat empty env. var as "undefined and use default()
+    result_dest_basedir: '{{ lookup("env","RESULTSDIR") |
+                             default(playbook_dir ~ "/results", True) }}'
+    result_filepaths: []  # do not use
+    _dstfnbuff: []  # do not use
+  tasks:
+    - name: The crio_integration_filepath is required
+      tags:
+          - integration
+      set_fact:
+        result_filepaths: "{{ result_filepaths + [crio_integration_filepath] }}"
+
+    - name: The crio_node_e2e_filepath is required
+      tags:
+          - e2e
+      set_fact:
+        result_filepaths: "{{ result_filepaths + [crio_node_e2e_filepath] }}"
```

Comment:
Tagging these tasks using the same ones from #927 to support running this under CI as separate jobs --or-- as a single-job that includes all hosts.  In other words, ``result_filepaths`` either ends up containing a single file path or two (or more if added later).